pkg/apis/core/v1/defaults_test.go

```diff
@@ -379,6 +379,571 @@ func testPodDefaults(t *testing.T, featuresEnabled bool) {
 	}
 }
 
+func TestPodResourcesDefaults(t *testing.T) {
+	cases := []struct {
+		name                     string
+		podLevelResourcesEnabled bool
+		containers               []v1.Container
+		podResources             *v1.ResourceRequirements
+		expectedPodSpec          v1.PodSpec
+	}{{
+		name:       "pod resources=unset, container resources=unset",
+		containers: []v1.Container{{Resources: v1.ResourceRequirements{}}, {Resources: v1.ResourceRequirements{}}},
+		expectedPodSpec: v1.PodSpec{
+			Containers: []v1.Container{{Resources: v1.ResourceRequirements{}}, {Resources: v1.ResourceRequirements{}}},
+		},
+	}, {
+		name: "pod resources=unset, container requests=unset limits=set",
```

Comment:
nit: `unset, limits=set`

Reply:
or maybe `;` as it's not acting on the noun container (i.e: it's pod + container limits set, not `pod resources=set; container requests=unset, limits=set`

Reply:
oh I see it's acting on the noun container. nevermind